expo/tests/contexts/act_techinfo_context.py

```diff
@@ -50,17 +50,18 @@ def __init__(self,
             event=self.sched_event,
             resource=ActResourceFactory(_item=self.act.actitem_ptr))
         if schedule_rehearsal:
-            self.rehearsal = _schedule_rehearsal(self.sched_event, self.act)
+            self.rehearsal = self._schedule_rehearsal(self.sched_event, self.act)
 
 
-def _schedule_rehearsal(s_event, act):
-    rehearsal = GenericEventFactory(type="Rehearsal Slot")
-    rehearsal_event = SchedEventFactory(eventitem=rehearsal.eventitem_ptr,
-                                        max_volunteer=10)
-    event_container = EventContainerFactory(
-        child_event=rehearsal_event,
-        parent_event=s_event)
-    ResourceAllocationFactory(resource=ActResourceFactory(
-        _item=act.actitem_ptr),
-                              event=s_event)
-    return rehearsal_event
+    def _schedule_rehearsal(self, s_event, act=None):
```

Comment:
I don't see where you're using the self. Why is this being moved into the class?